conda_smithy/lint_recipe.py

```diff
@@ -1181,8 +1181,24 @@ def run_conda_forge_specific(meta, recipe_dir, lints, hints):
         # Check if all maintainers have either commented or are the PR author
```

Comment:
Seeing this code now, I wonder if we should also get the emoji reactions in the comments Sometimes folks just leave a 👍 as the confirmation.

I think this could work (for the block above):

```python
commenters.update(
    {
        reaction.user.login 
        for comment in issue comments
        for reaction in issue.get_reactions()
        if reaction.content in ("+1", "heart", "rocket", "hooray")
    }
)
```

Refs:
- https://pygithub.readthedocs.io/en/latest/github_objects/IssueComment.html#github.IssueComment.IssueComment.get_reactions
- https://pygithub.readthedocs.io/en/latest/github_objects/Reaction.html
- https://docs.github.com/en/rest/reactions/reactions?apiVersion=2022-11-28#about-reactions

Reply:
I think that's a good idea! My memory is a little vague but I think the problem during implementation was that these features weren't available yet.

Reply:
@jaimergp do you mind sending a PR?